src/applications/gi-sandbox/containers/TuitionAndHousingEstimates.jsx

```diff
@@ -93,6 +93,11 @@ export function TuitionAndHousingEstimates({
           setOnlineClasses(e.target.value);
         }}
       />
+      {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
+      <div tabIndex="0" className="vads-u-padding-top--2">
```

Comment:
@ksantiagoBAH what are we using this tabindex for? It doesn't seem like it's something that needs to be focusable. If it needs programmatic focus (like an expanding box or something), we would set that value to `-1` so it doesn't interfere with normal keyboard usage

Reply:
It was requested in a 508 defect by @noahgelman  for screen reader users to announce that changes that will occur.

Reply:
This wouldn't force an announcement, just interrupt the tab order. There are some best practices for things like this here: https://www.w3.org/WAI/tutorials/forms/instructions/ namely `aria-labelledby` or `aria-describedby`

Reply:
@timwright12 The content that they are adding tabindex to is not information related to any specific field. It is a general note that describes an important functionality of the page that we want screen reader users to not miss as they would be tabbing through the field. It's more akin to a warning or alert which a tabindex is sometimes added to make sure it's tabbed to.

Reply:
@noahgelman I'll defer to your expertise, but wouldn't this be more like an aria-describedby on the submit button in that were the case or maybe set to the form itself?

```
<form aria-describedby="thenote">
  <p id="thenote"></p>
```

Reply:
@timwright12 I think putting it on the submit button should be fine